samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Startup.cs

```diff
@@ -75,7 +75,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton(skillsConfig);
 
             // Register AuthConfiguration to enable custom claim validation.
-            services.AddSingleton(sp => new AuthenticationConfiguration { ClaimsValidator = new AllowedCallersClaimsValidator(skillsConfig) });
+            var allowedCallers = (from skill in skillsConfig.Skills.Values select skill.AppId).ToList();
+            services.AddSingleton(sp => new AuthenticationConfiguration { ClaimsValidator = new Microsoft.Bot.Connector.Authentication.AllowedCallersClaimsValidator(allowedCallers) });
```

Comment:
```suggestion
            services.AddSingleton(sp => new AuthenticationConfiguration { ClaimsValidator = new AllowedCallersClaimsValidator(allowedCallers) });
```

As we removed the one of the project, it's not necessary to keep that reference